### PR TITLE
[INLONG-7332][Manager] Change the field length in SQL files

### LINKED ISSUE #7332: [Improve][Manager] Improve the field length in SQL files
### Description

some field type define is not enough

`inlong_stream_field#field_format`
`inlong_stream_field#meta_field_name`

`stream_source_field#field_format`
`stream_source_field#meta_field_name`

`stream_transform_field#field_format`
`stream_transform_field#meta_field_name`

`stream_sink_field#field_format`
`stream_sink_field#source_field_name`
`stream_sink_field#meta_field_name`

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- [INLONG-7332][Manager] Improve sql script field type define

- Fixes #7332 

### Motivation

* Some field type is not enough for service, So I modify type

### Modifications

* Modify field type of four table

### CODE REVIEW

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 1]
```diff
@@ -298,8 +298,8 @@ CREATE TABLE IF NOT EXISTS `inlong_stream_field`
     `field_type`          varchar(20)  NOT NULL COMMENT 'field type',
     `field_comment`       varchar(50)  DEFAULT NULL COMMENT 'Field description',
     `is_meta_field`       smallint(3)  DEFAULT '0' COMMENT 'Is this field a meta field? 0: no, 1: yes',
-    `meta_field_name`     varchar(20)  DEFAULT NULL COMMENT 'Meta field name',
-    `field_format`        varchar(50)  DEFAULT NULL COMMENT 'Field format, including: MICROSECONDS, MILLISECONDS, SECONDS, SQL, ISO_8601 and custom such as yyyy-MM-dd HH:mm:ss',
+    `meta_field_name`     varchar(120)  DEFAULT NULL COMMENT 'Meta field name',
+    `field_format`        text  DEFAULT NULL COMMENT 'Field format, including: MICROSECONDS, MILLISECONDS, SECONDS, SQL, ISO_8601 and custom such as yyyy-MM-dd HH:mm:ss',
```
Comment (healchow) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:302]: Why does `field_format` need text type? In what scenario does it store long strings?

Comment (gong) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql:302]: > Why does `field_format` need text type? In what scenario does it store long strings?

`field_format` can store serialize format of complex type. It cloud be too long. So I set text.

Comment (healchow) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql:302]: Okay, please add this scenario to the comment info, thanks.

Comment (gong) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql]: done

**Review by healchow — APPROVED** [review]

**Comment by vernedeng** [comment]
It's better to provide a upgrate SQL for each modifcatition of table defination.

**Review by fuweng11 — APPROVED** [review]

**Comment by gong** [comment]
> It's better to provide a upgrate SQL for each modifcatition of table defination.

ok

File: inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql [hunk 2]
```diff
@@ -319,29 +319,29 @@ CREATE TABLE IF NOT EXISTS `operation_log`
...
-    `ext_params`          mediumtext            DEFAULT NULL COMMENT 'Another fields will be saved as JSON string, such as filePath, dbName, tableName, etc',
-    `version`             int(11)               DEFAULT '1' COMMENT 'Stream source version',
-    `status`              int(4)                DEFAULT '110' COMMENT 'Stream source status',
-    `previous_status`     int(4)                DEFAULT '110' COMMENT 'Previous status',
-    `is_deleted`          int(11)               DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, > 0: deleted',
-    `creator`             varchar(64)  NOT NULL COMMENT 'Creator name',
-    `modifier`            varchar(64)           DEFAULT NULL COMMENT 'Modifier name',
-    `create_time`         timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Create time',
-    `modify_time`         timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Modify time',
+    `id`                        int(11)      NOT NULL AUTO_INCREMENT COMMENT 'ID',
```
Comment (dockerzhang) [inline inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql:322]: it's better to keep the same indent.

File: inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql [hunk 3]
```diff
@@ -750,11 +750,11 @@ CREATE TABLE IF NOT EXISTS `stream_heartbeat`
 -- ----------------------------
 CREATE TABLE IF NOT EXISTS `audit_base`
 (
-    `id`               int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
-    `name`             varchar(256) NOT NULL COMMENT 'Audit base name',
-    `type`             varchar(20)  NOT NULL COMMENT 'Audit base item type, such as: AGENT, DATAPROXY, etc',
-    `is_sent`          int(4)       NOT NULL DEFAULT '0' COMMENT '0: received, 1: sent',
-    `audit_id`         varchar(11)  NOT NULL COMMENT 'Audit ID mapping of audit name',
+    `id`       int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
+    `name`     varchar(256) NOT NULL COMMENT 'Audit base name',
```
Comment (dockerzhang) [inline inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql:754]: ditto

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 4]
```diff
@@ -336,29 +336,29 @@ CREATE TABLE IF NOT EXISTS `operation_log`
...
-    `ext_params`          mediumtext            DEFAULT NULL COMMENT 'Another fields will be saved as JSON string, such as filePath, dbName, tableName, etc',
-    `version`             int(11)               DEFAULT '1' COMMENT 'Stream source version',
-    `status`              int(4)                DEFAULT '110' COMMENT 'Stream source status',
-    `previous_status`     int(4)                DEFAULT '110' COMMENT 'Previous status',
-    `is_deleted`          int(11)               DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, > 0: deleted',
-    `creator`             varchar(64)  NOT NULL COMMENT 'Creator name',
-    `modifier`            varchar(64)           DEFAULT NULL COMMENT 'Modifier name',
-    `create_time`         timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Create time',
-    `modify_time`         timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Modify time',
+    `id`                        int(11)      NOT NULL AUTO_INCREMENT COMMENT 'ID',
```
Comment (dockerzhang) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:339]: ditto

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 5]
```diff
@@ -794,11 +794,11 @@ CREATE TABLE IF NOT EXISTS `stream_heartbeat`
 -- ----------------------------
 CREATE TABLE IF NOT EXISTS `audit_base`
 (
-    `id`               int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
-    `name`             varchar(256) NOT NULL COMMENT 'Audit base name',
-    `type`             varchar(20)  NOT NULL COMMENT 'Audit base item type, such as: AGENT, DATAPROXY, etc',
-    `is_sent`          int(4)       NOT NULL DEFAULT '0' COMMENT '0: received, 1: sent',
-    `audit_id`         varchar(11)  NOT NULL COMMENT 'Audit ID mapping of audit name',
+    `id`       int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
```
Comment (dockerzhang) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:797]: ditto

File: inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql]: reverted.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**